department-of-veterans-affairs/caseflow
# Queue | Edit issue diagnostic code

Resolves #5202 

### Description
Within the `AddEditIssueView`, we previously didn't allow users to set diagnostic codes (i.e. `8312: Hypoglossal nerve neuritis`)

### Acceptance Criteria 
- [x] Confirm edits made to issue diagnostic codes persist after saving
- [x] Diagnostic code dropdown should only be enabled in cases described by [Appendix F - VACOLS Issue Coding.pdf](https://git.io/vxBtN)

### Testing Plan
1. Go to Draft Decision checkout flow, add or edit issue, set diagnostic code
2. Save edits, confirm code is updated in SelectDispositionsView, AppealDetail
